.github/workflows/release.yaml

```diff
@@ -135,6 +135,7 @@ jobs:
         run: |
           make build_cli
           cp "secretcli" "secretcli-$RUNNER_OS"
+          make build_linux_arm64_cli
```

Comment:
Copying from the other comment:

The arm64 build was added to a line that runs on each build target in the matrix. A better way is to add the target to the matrix instead, that way it gets built and uploaded/released automatically. IMO linux-arm64 is supported, and there should be a mac m1 (darwin-arm64) target supported on GH actions, though I'm not 100% about that